components/common/listingCart/ListingCartItem.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="py-2 px-6">
+    <div class="py-2 listing-border" />
+    <div class="is-flex is-justify-content-space-between">
+      <div class="is-flex">
+        <div>
+          <BasicImage
+            :src="avatar"
+            :alt="nft?.name"
+            class="border image is-48x48" />
+        </div>
+
+        <div
+          class="is-flex is-flex-direction-column is-justify-content-space-between ml-4 limit-width">
+          <div
+            class="has-text-weight-bold has-text-color line-height-1 no-wrap is-clipped is-ellipsis">
+            {{ nft.name }}
+          </div>
+          <div class="line-height-1 no-wrap is-clipped is-ellipsis">
+            {{ nft.collection?.name || nft.collection.id }}
+          </div>
+        </div>
+      </div>
+
+      <NeoButton
+        class="has-text-grey pt-4"
+        variant="text"
+        no-shadow
+        icon="trash"
+        icon-pack="far"
+        @click.native="listingCartStore.removeItem(nft.id)" />
+    </div>
+    <div class="is-flex is-justify-content-space-between pt-2">
+      <div class="is-flex is-flex-direction-column">
+        <div class="has-text-grey"><small>Floor</small></div>
+        <span>{{ floor }}</span>
+      </div>
+
+      <div class="is-flex is-align-items-end pt-2">
+        <ListingCartPriceInput
+          v-model="listingCartStore.getItem(nft.id).listPrice"
+          style="height: 100%" />
```

Comment:
```suggestion
class="h-full" />
```

Reply:
removed it entirely since it is not needed anymore.